content/embeds/hardware-requirements-embed.md

```diff
@@ -23,15 +23,17 @@ We recommend these hardware requirements for production systems or for developme
 
 | Item | Description | Minimum Requirements | Recommended |
 |------------|-----------------|------------|-----------------|
-| Nodes per cluster | At least 3 nodes are required to support a reliable, highly available deployment that handles process failure, node failure, and network split events in a consistent manner. | 3 nodes | >= 3 nodes (Must be an odd number of nodes) |
+| Nodes per cluster<sup>*</sup> | At least 3 nodes are required to support a reliable, highly available deployment that handles process failure, node failure, and network split events in a consistent manner. | 3 nodes | >= 3 nodes (Must be an odd number of nodes) |
 | Cores<sup>*</sup> per node | RS is based on a multi-tenant architecture and can run multiple Redis processes (or shards) on the same core without significant performance degradation. | 4 cores | >=8 cores |
 | RAM<sup>*</sup> per node | Defining your RAM size must be part of the capacity planning for your Redis usage. | 15GB | >=30GB |
 | Ephemeral Storage | Used for storing [replication files (RDB format) and cluster log files]({{< relref "/rs/administering/designing-production/persistent-ephemeral-storage.md" >}}). | RAM x 2 | >= RAM x 4 |
 | Persistent Storage | Used for storing [snapshot (RDB format) and AOF files]({{< relref "/rs/administering/designing-production/persistent-ephemeral-storage.md" >}}) over a persistent storage media, such as AWS Elastic Block Storage (EBS) or Azure Data Disk. | RAM x 3 | In-memory >= RAM x 6 (except for [extreme 'write' scenarios]({{< relref "/rs/administering/designing-production/performance/disk-sizing-heavy-write-scenarios.md" >}})); [Redis on Flash]({{< relref "/rs/concepts/memory-architecture/redis-flash.md" >}}) >= (RAM + Flash) x 5. |
 | Network | We recommend using multiple NICs per node where each NIC is >100Kbps, but RS can also run over a single 1Gbps interface network used for processing application requests, inter-cluster communication, and storage access. | 1G | >=10G |
 
 <sup>*</sup>Additional considerations:
-
+- Nodes per Cluster:
+    - For clusters which are configured with 3 nodes, where one of them is considered as a quorum node - database availability can be compromised in a (non-quorum) node failure scenario.
```

Comment:
```suggestion
    - When a cluster has three nodes and one of those node is considered a quorum node, database availability can be compromised when a node fails for non-quorum related reasons. 
```

Reply:
not exactly, it's when a node, which is not the quorum node, fails.
support call it "data-node" but I don't think that's an official term we use

Reply:
Alon, IMHO "non-quorum node" should be fine in this case.
Something like:

For clusters consisting of only three nodes, with one of them configured as a quorum node - database redundancy can be compromised if a single non-quorum node fails.

I also switched "availability" to "redundancy", since the DB should be still accessible. Not sure whether we use this term in our docs, though.